### PR TITLE
Update README.md

### LINKED ISSUE #5045: Past preview builds archive
Is there a list or archive of past preview builds? For example, links all point to latest which is preview5 at this point but I'd like to get preview 4.

### PR DESCRIPTION
**Pull request by blackdwarf**
Add information about the Preview 4 bits.

Fixes #5045, #5009

skip ci please

/cc @piotrpMSFT @livarcocc @srivatsn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,23 @@
 
 This repo contains the source code for cross-platform [.NET Core](http://github.com/dotnet/core) command line toolchain. It contains the implementation of each command, the native packages for various supported platforms as well as documentation. 
 
-RTM and Preview 2 bits
----------------------
-To get the latest released bits (RTM for .NET Core and Preview 2 for tooling), 
-check out our [Getting started page](http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409).
+Preview 4 release - MSBuild based tools
```
Comment (livarcocc) [inline README.md]: @blackdwarf Should we add information about global.json and sdk version in case people want to run preview2 and preview4 side-by-side?

Comment (blackdwarf) [inline README.md]: @livarcocc good idea.

**Comment by srivatsn** [comment]
@blackdwarf No need to tag me for non-product changes.

**Comment by blackdwarf** [comment]
@srivatsn got it, sorry for the trouble.

**Comment by srivatsn** [comment]
No worries. Said that to save you time in the future.

**Pull request merged**